Project 2/four-card-feature/style.css

```diff
@@ -0,0 +1,203 @@
+body {
+  font-size: 15px;
+  background-color: hsl(0, 0%, 98%);
+  font-family: "Poppins", sans-serif;
+  margin: 0;
+  border: 0;
+  width: fit-content;
+}
+
+.container {
+  max-width: 1440px;
+  margin: 50px auto;
+}
+
+p {
+  color: hsl(229, 6%, 66%);
+}
+
+.SectionA {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.SectionB {
+  display: flex;
+  justify-content: space-around;
+  height: 550px;
+  width: 1200px;
+}
+
+.secB {
+  display: flex;
+  justify-content: center;
+}
+
+.Reliable {
+  color: hsl(234, 12%, 34%);
+  font-weight: 400;
+  font-size: 28px;
+}
+
+.Powered {
+  color: hsl(234, 12%, 34%);
+  font-weight: 600;
+  font-size: 28px;
+}
+
+.our {
+  color: hsl(229, 6%, 66%);
+  font-size: 18px;
+}
+
+h1 {
+  font-size: 20px;
+  color: hsl(234, 12%, 34%);
+}
+
+.supervisor {
+  border-top: 5px solid hsl(180, 62%, 55%);
+  border-radius: 3%;
+  background-color: white;
+  -webkit-box-shadow: 4px 16px 34px -16px rgba(181, 181, 191, 1);
+  -moz-box-shadow: 4px 16px 34px -16px rgba(181, 181, 191, 1);
+  box-shadow: 4px 16px 34px -16px rgba(181, 181, 191, 1);
+  width: 300px;
+  height: 35%;
```

Comment:
to może być ryzykowne. Jeżeli już musimy ustawiać height na twardo, to raczej wartościami px

Reply:
Ok.